app/index.sass

```diff
@@ -2,10 +2,11 @@
 $icon-font-path: '../fonts/'
 
 // bower:scss
+// endbower
 @import 'bower_components/bootswatch-scss/flatly/_variables.scss'
 @import 'bower_components/bootstrap-sass-official/assets/stylesheets/_bootstrap.scss'
```

Comment:
we acutally end up need bootstrap-sass-official, as bootswatch relies on it and just overrides the scss file. One option is to just copy the file to bootswatch, and then remove the bootstrap dependency.